src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,116 @@
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            while (resultSet.next()) {
+                manufacturer.setId(resultSet.getLong(1));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer to database "
+                    + manufacturer, e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String select = "SELECT * FROM manufacturer WHERE manufacturer_id = ? AND "
+                + "manufacturer_deleted = false;";
+        Manufacturer manufacturer = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(select)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = getManufacturer(resultSet);
+            }
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't find manufacturer by id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String selectAll = "SELECT * FROM manufacturer WHERE manufacturer_deleted = false;";
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(selectAll)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufactorers from database", e);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String update = "UPDATE manufacturer SET manufacturer_name = ?, manufacturer_country = ? "
+                + "WHERE manufacturer_id = ? AND manufacturer_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(update)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.setLong(3, manufacturer.getId());
+            statement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer: " + manufacturer, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String delete = "UPDATE manufacturer SET manufacturer_deleted = true WHERE "
```

Comment:
```suggestion
        String delete = "UPDATE manufacturer SET deleted = true WHERE "
```
please check your columns naming